### PR TITLE
Export status of Data Security At Rest from EVE to EVC

### PR DESCRIPTION
**Pull request by cshari-zededa**
Signed-off-by: Hariharasubramanian C S <cshari@zededa.com>

### CODE REVIEW

File: api/proto/info/info.proto [hunk 1]
```diff
@@ -183,6 +183,18 @@ enum HwSecurityModuleStatus {
...
+enum DataSecAtRestStatus {
+    DATASEC_AT_REST_UNKNOWN  = 0; // Status is unknown
+    DATASEC_AT_REST_DISABLED = 1; // Enabled, but not being used
+    DATASEC_AT_REST_ENABLED = 2;  // Enabled, and used
+    DATASEC_AT_REST_ERROR = 4;    // Enabled, but encountered an error
+}
+
+message DataSecAtRest {
+    DataSecAtRestStatus status = 1; // Current status
+    string info = 2;                // Informational string augmenting status
```
Comment (eriknordmark) [inline api/proto/info/info.proto]: Do you expect this to be used for anything but errors?

For errors we have an existing (errorInfo?) message which has a string and a timestamp for when the error occurred. Don't know if that would make sense here.

Comment (cshari-zededa) [reply api/proto/info/info.proto]: Hi Erik,
I am planning to have this field as a more verbose form of the state. i.e. even if this is enabled/disabled state, we can add more info in the form a string. E.g. if enabled, then give the list of default vaults etc..

Thanks
Hari

Comment (cshari-zededa) [reply api/proto/info/info.proto]: e.g. if disk encryption feature is available but not being used (Not an error exactly), we can give a more verbose string like "file system incompatibility - needs disruptive usb upgrade".

Comment (eriknordmark) [reply api/proto/info/info.proto]: OK, there might be some UX questions around that when it comes to localization, but we can add a detailed enum later if that makes L10N easier.

**Pull request merged**